sys/include/net/dhcpv6/client.h

```diff
@@ -170,6 +199,44 @@ void dhcpv6_client_conf_prefix(unsigned netif, const ipv6_addr_t *pfx,
                                unsigned pfx_len, uint32_t valid,
                                uint32_t pref);
 
+/**
+ * @brief   Checks if the given network interface is configured
+ *          to use DHCPv6 IA_NA
+ *
+ * @param[in] netif     Network interface to check.
+ *
+ * @return  true, if the network interface is set up for IA_NA.
+ */
+bool dhcpv6_client_check_ia_na(unsigned netif);
+
+/**
+ * @brief   Configures a address lease that is provided by the server.
+ *
+ * @param[in] netif     Network interface the address was for.
+ * @param[in] addr      The assigned address.
+ *
+ * @return <0 on error.
```

Comment:
Oops, that was my error: the success case should still be documented too of course.

Reply:
Oh, I see! I'll add it back in :)

Reply:
Done in 540ab0bbfa19705e904eeac205d5b4f2354e7b9d